app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt

```diff
@@ -124,7 +124,7 @@ class TopicLessonsFragmentTest {
 
   private fun setUpTestApplicationComponent() {
     ApplicationProvider.getApplicationContext<TestApplication>()
-      .inject(topicLessonsFragmentTest = this)
+      .inject(this)
```

Comment:
This can be accommodated on a single line (as in the current codebase).

Reply:
@Sci3fic please reply to all comments.